src/core/js/models/adaptModel.js

```diff
@@ -555,38 +558,32 @@ define([
         this._hasSiblingsAndSelf = true;
       }
 
-      var siblingsCollection = new Backbone.Collection(siblings);
+      const siblingsCollection = new Backbone.Collection(siblings);
       this.set('_siblings', siblingsCollection);
       return siblingsCollection;
-    },
-
-    setOnChildren: function (key, value, options) {
+    }
 
-      var args = arguments;
+    setOnChildren(key, value, options) {
```

Comment:
Suggest `setOnChildren(...args)` or similar? Would avoid the linter complaining about variables that are never used.

Reply:
Hmmm...